### PR TITLE
Search across workspace related changes

### PR DESCRIPTION
**Pull request by rohi-modak-ct**

### CODE REVIEW

File: workspace_management/app/db_interfaces/organization.py [hunk 1]
```diff
@@ -70,9 +70,3 @@ def update(self, favorite_type, favorite_id, active):
                                                                     favourite_attr_type=favorite_type,
                                                                     favourite_attr_id=favorite_id,
                                                                     defaults={"is_active": active})
-
-
-def update_favorite(action, data):
```
Comment (agarwalgazal) [inline workspace_management/app/db_interfaces/organization.py:75]: why is this removed from here

Comment (rohi-modak-ct) [reply workspace_management/app/db_interfaces/organization.py:75]: We are not using this anymore as we are using the db interface class directly. So, this has been removed
